pytest/tests/loadtest/locust/common/sharding.py

```diff
@@ -256,7 +256,14 @@ def random_prefix_between(lower, upper, free_length=6):
     return prefix
 
 
-def random_account_between(base_name, suffix, lower, upper, free_length=6):
+# Maximum length of AccountIds (https://nomicon.io/DataStructures/Account).
+MAX_NEAR_ACCOUNT_ID_LENGTH = 64
+
+
+def random_account_between(base_name, suffix, lower, upper):
+    free_length = MAX_NEAR_ACCOUNT_ID_LENGTH - len(base_name) - len(suffix) - 1
```

Comment:
wouldn't it be better to do something like this?

```diff
diff --git a/pytest/tests/loadtest/locust/common/sharding.py b/pytest/tests/loadtest/locust/common/sharding.py
index 68576f080..741383796 100644
--- a/pytest/tests/loadtest/locust/common/sharding.py
+++ b/pytest/tests/loadtest/locust/common/sharding.py
@@ -261,7 +261,7 @@ MAX_NEAR_ACCOUNT_ID_LENGTH = 64
 
 
 def random_account_between(base_name, suffix, lower, upper):
-    free_length = MAX_NEAR_ACCOUNT_ID_LENGTH - len(base_name) - len(suffix) - 1
+    free_length = min(6, MAX_NEAR_ACCOUNT_ID_LENGTH - len(base_name) - len(suffix) - 1)
     assert free_length > 0, f"""No space left for random prefix. Check base_name={
         base_name} and suffix={suffix}"""
     prefix = random_prefix_between(lower, upper, free_length)
```

otherwise if you print out the account IDs created in the unit test below, you get stuff like this:

```
l23ku8ku7qdfofz5f2ai9766bz4fejmfhtb8l12fzoi67ni9vaj4_ft.foo.near
pikpcdtw12ybe1cstlo2sos20nogm63ns71y7k7fxu8vahbssnpu_ft.foo.near
iwxy1rrvevuvmo09oqzzygl12wiloe0sdnbmwbim1kftnv19g743_ft.foo.near
ech1wpchli0hbgog20tzigmlbavoswach92gzqlt22qsa6550xav_ft.foo.near
...
```

which seems not so ideal, and could make working with those accounts more difficult, since then you can't create any subaccounts of the accounts generated by this function

Reply:
good catch. done.